mkdocs/docs/api.md

```diff
@@ -292,6 +292,39 @@ The nested lists indicate the different Arrow buffers, where the first write res
 
 <!-- prettier-ignore-end -->
 
+### Add Files
+
+Expert Iceberg users may choose to commit existing parquet files to the Iceberg table as data files, without rewriting them.
```

Comment:
Shall we mention in the doc that this procedure currently only work for unpartitioned table?

Reply:
Maybe! We've already discussed the different approaches for supporting adds to partitioned tables extensively, so I'm optimistic we'll get it in before the next release. I'll put it up shortly after this is merged.

Reply:
Sounds great! Thanks!